tests/Microsoft.Identity.Test.Common/TestConstants.cs

```diff
@@ -31,7 +31,7 @@ public static HashSet<string> s_scope
         public static readonly string[] s_graphScopes = new[] { "user.read" };
         public const uint JwtToAadLifetimeInSeconds = 60 * 10; // Ten minutes
         public const string ClientCredentialAudience = "https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/v2.0";
-        public const string AutomationTestThumbprint = "444B697D869032F29F9A162D711AF3E2791AD748";
+        public const string AutomationTestCertName = "AzureADIdentityDivisionTestAgentCert";

```

Comment:
if the lab access cert `LabAccessCertName` is used for all tests, no need to store it on `AutomationTestCertName` as well. Let's just use a common name to refer the same cert

Reply:
AutomationTestCertName is used at more than one places. Removed LabAccessCertName